### PR TITLE
add script

### PR DESCRIPTION
**Pull request by paulzay**

### CODE REVIEW

File: scripts/00150-6lackHat.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def Hello(full_name, ID, email, language):
+    return "Hello World, this is {} with HNGi7 ID {} and email address {} using {} for stage 2 task".format(full_name, ID, email, language)
```
Comment (ifatoki) [inline scripts/00150-6lackHat.py]: Hi, this format doesn't align with the requirements of the team. The email should come after the word `task` and a space. For more information check [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700)

**Review by ifatoki — CHANGES_REQUESTED** [review]

File: scripts/00150-6lackHat.py (hunk 1 above)
Comment (paulzay) [reply scripts/00150-6lackHat.py]: Hey my script is yet to be added

Comment (lindaokorie27) [reply scripts/00150-6lackHat.py]: You still haven't added the email after the word task. Look at other people's output [here](https://github.com/ifatoki/hng-wonderwoman-task-1/pull/253)

File: scripts/00150-6lackHat.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+def Hello(full_name, ID, language, email):
+    return "Hello World, this is {} with HNG ID {} using {} for stage 2 task".format(full_name, ID, language,email)
```
Comment (ifatoki) [inline scripts/00150-6lackHat.py]: Please check this. Your email should come after the word 'task'. Right now, that is not done

**Review by ifatoki — CHANGES_REQUESTED** [review]

File: scripts/00150-6lackHat.py (hunk 2 above)
Comment (paulzay) [reply scripts/00150-6lackHat.py]: done

**Review by lindaokorie27 — APPROVED** [review]

**Review by ifatoki — APPROVED** [review]

**Pull request merged**